naloge/topo.tex

```diff
@@ -15,6 +15,111 @@
 \podnaslov{Graf}
 \end{slika}
 \end{vprasanje}
-\begin{odgovor}
+\begin{odgovor}\\
+
+\noindent a) Za lažjo predstavo vozlišč v sledečih algoritmih
+označimo črke A do H, ki označujejo vozlišča s števili od 0 do 7.\\
+
+\noindent Upoštevamo, da je graf acikličen, torej za vsaki vozliščii $u, v \in G$ velja: \\[5px]
+$u \rightarrow v \implies $ pooznaka(u)\ \textgreater \ pooznaka(v), \\[5px]
+kjer je pooznaka števec s katerim označimo vozlišče, ko ga v algoritmu dfs docela preiščemo.\\
+
+\noindent Trditev lahko dokažemo s premislekom poteka določitev pooznak,
+ko pridemo v algoritmu dfs prvo do u in prvo do v vozlišč.
+Trditev nam omogoča zapis krajšega algoritma, ki bo implementiral dfs.
+Ideja je, da padajoče uredimo vozlišča po njihovih pooznakah,
+po čemer dobimo inverzno topološko ureditev.
+To pa dosežemo tako, da po tem, ko docela preiščemo vozlišče, tega položimo na sklad. \\
+
+\begin{algorithmic}
+\Function{Topo}{V, E}
```

Comment:
```suggestion
\Function{Topo}{$G = (V, E)$}
```
Tako bo jasno, da je vhod graf, samo na *V* in *E* se lahko potem posebej sklicuješ.